adapters/oidc/js/src/keycloak.js

```diff
@@ -527,9 +527,32 @@ function Keycloak (config) {
         }
 
         var access = kc.resourceAccess[resource || kc.clientId];
-        return !!access && access.roles.indexOf(role) >= 0;
+        if (!Array.isArray(obj))
+           return !!access && access.roles.indexOf(role) >= 0;
+        } else if (!!access) {
+           return roles.some(function(role) {
+               return access.roles.indexOf(role) >= 0;
+           });
+        } else {
+           return false;
+        }
     }
 
+//     kc.hasAtLeastOneResourceRoles = function(resource, ...roles) {
+//            if (!kc.resourceAccess) {
+//                return false;
+//            }
+//
+//            var access = kc.resourceAccess[resource || kc.clientId];
+//            if (!!access) {
+//                return roles.some(function(role) {
+//                    return access.roles.indexOf(role) >= 0;
+//                });
+//            } else {
+//               return false;
+//            }
+//        }
+
```

Comment:
Let's not add commented out code.
```suggestion
```

Reply:
Sorry, it is temporary. 
Although Keycloak have been working with this code, some tests related to ui were failing.
I try in another way. Finnally, I will remove it.